components/button/button-icon.js

```diff
@@ -20,17 +20,18 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 			/**
 			 * Aligns the leading edge of text if value is set to "text"
 			 * @type {'text'|''}
-			 * @default ""
 			 */
 			hAlign: { type: String, reflect: true, attribute: 'h-align' },
 
 			/**
 			 * REQUIRED: Preset icon key (e.g. "tier1:gear")
+			 * @type {string}
```

Comment:
`type` will only get to string in `custom-elements.json` if the value is actually initialized in the constructor, otherwise we need to set `@type` here